src/prepack-cli.js

```diff
@@ -475,6 +562,14 @@ fi
         return;
       }
       let serialized = prepackFileSync(inputFilenames, resolvedOptions);
+      if (reproUnconditionally) {
+        if (serialized.sourceFilePaths) {
+          generateDebugRepro(serialized.sourceFilePaths.sourceMaps, serialized.sourceFilePaths.sourceFiles);
+        } else {
+          generateDebugRepro([], []);
```

Comment:
Why are we generating empty zip?

Reply:
Inputs without sourcemaps won't have a sourcemap or sourcefiles, but we still can zip in the input files. The input filenames were saved when reading CLI arguments, and thus don't need to be passed in. Will clarify in a comment